mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -81,7 +83,7 @@ def run():
         line_freq = None if opt.line_freq == "None" else opt.line_freq
         raw.info['line_freq'] = line_freq
     write_raw_bids(raw, bids_path, event_id=opt.event_id,
-                   events_data=opt.events_data, overwrite=opt.overwrite,
+                   events=opt.events, overwrite=opt.overwrite,
```

Comment:
This will break backward compat if you don’t pass events_data no?

Reply:
Addressed in [f6de206](https://github.com/mne-tools/mne-bids/pull/1054/commits/f6de2065f60f52e4cf4fc03606da636cdcd831f8)